### PR TITLE
shippable run against all stable and devel ansible

### PR DESCRIPTION
**Pull request by maxamillion**
Signed-off-by: Adam Miller <admiller@redhat.com>

##### SUMMARY

shippable run against all stable and devel ansible

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

ansible.posix

### CODE REVIEW

File: shippable.yml [hunk 1]
```diff
@@ -8,21 +8,51 @@ matrix:
...
-    - env: T=linux/centos6/1
-    - env: T=linux/centos7/1
-    - env: T=linux/centos8/1
-    - env: T=linux/fedora30/1
-    - env: T=linux/fedora31/1
-    - env: T=linux/opensuse15py2/1
-    - env: T=linux/opensuse15/1
-    - env: T=linux/ubuntu1604/1
-    - env: T=linux/ubuntu1804/1
+    - env: T=2.9/aix/7.2/1
```
Comment (gundalow) [inline shippable.yml:11]: What's happened to `sanity` and `units`?

File: tests/utils/shippable/shippable.sh [hunk 2]
```diff
@@ -68,47 +106,81 @@ else
...
+                    # remove numbered component from stub files when converting to tags
+                    flags="${flags//stub-[0-9]*/stub}"
+                    flags="${flags//=/,}"
+                    flags="${flags//[^a-zA-Z0-9_,]/_}"
+
+                    bash <(curl -s https://codecov.io/bash) \
+                        -f "${file}" \
+                        -F "${flags}" \
+                        -n "${test}" \
+                        -t 20636cf5-4d6a-4b9a-8d2d-6f22ebbaa752 \
```
Comment (gundalow) [inline tests/utils/shippable/shippable.sh:162]: ansible.posix's token is:
```suggestion
                        -t 54c9aa8f-073c-47b2-87e5-b756f5123ccc \
```

File: tests/utils/shippable/shippable.sh [hunk 3]
```diff
@@ -26,11 +33,42 @@ fi
 command -v python
 python -V
 
+function retry
```
Comment (gundalow) [inline tests/utils/shippable/shippable.sh:36]: `retry` isn't currently needed, though I guess having this here makes it easier to keep the files in sync across repos.

File: tests/utils/shippable/shippable.sh [hunk 4]
```diff
@@ -26,11 +33,42 @@ fi
...
+fi
+
+export ANSIBLE_COLLECTIONS_PATHS="${HOME}/.ansible"
+SHIPPABLE_RESULT_DIR="$(pwd)/shippable"
+TEST_DIR="${ANSIBLE_COLLECTIONS_PATHS}/ansible_collections/ansible/posix"
+mkdir -p "${TEST_DIR}"
+cp -aT "${SHIPPABLE_BUILD_DIR}" "${TEST_DIR}"
+cd "${TEST_DIR}"
+
+# START: HACK install dependencies
```
Comment (gundalow) [inline tests/utils/shippable/shippable.sh:69]: Needed for integration test dependencies
```suggestion
# START: HACK install dependencies
retry ansible-galaxy collection install community.general
```

File: shippable.yml (hunk 1 above)
Comment (maxamillion) [reply shippable.yml:11]: @gundalow thos are running in Zuul now, we're aiming to move everything to Zuul before Shippable is sunset by JFrog but still leveraging Shippable for now for a lot of stuff until we have time to transition.

File: shippable.yml [hunk 5]
```diff
@@ -8,21 +8,51 @@ matrix:
...
+    - env: T=2.9/linux/fedora30/1
+    - env: T=2.9/linux/fedora31/1
+    - env: T=2.9/linux/opensuse15py2/1
+    - env: T=2.9/linux/opensuse15/1
+    - env: T=2.9/linux/ubuntu1604/1
+    - env: T=2.9/linux/ubuntu1804/1
+    - env: T=2.10/aix/7.2/1
+    - env: T=2.10/osx/10.11/1
+    - env: T=2.10/rhel/7.6/1
+    - env: T=2.10/rhel/8.1/1
```
Comment (felixfontein) [inline shippable.yml:29]:
```suggestion
    - env: T=2.10/rhel/8.2/1
```
Same for devel.

**Comment by maxamillion** [comment]
recheck

File: shippable.yml (hunk 1 above)
Comment (gundalow) [reply shippable.yml:11]: Cool, thanks.

**Pull request merged**